coderd/database/migrations/000243_notifications_template_manual_build_failure.up.sql

```diff
@@ -0,0 +1,26 @@
+INSERT INTO
+    notification_templates (
+        id,
+        name,
+        title_template,
+        body_template,
+        "group",
+        actions
+    )
+VALUES (
+        '48a9d2b9-3655-430c-a31a-2442479e7519',
+        'Template Manual Build Failure',
+        E'Workspace with template "{{.Labels.name}}" failed to build',
+        E'Hi {{.UserName}}\n\nThe workspace **{{.Labels.workspaceName}}**, using the template **{{.Labels.name}}**, failed during a manual build({{.Labels.transition}}) initiated by the user **{{.Labels.initiator}}**.',
```

Comment:
> This sentence is a bit too complex. Can we break this up?

Sure, but we should be cautious about breaking it down too much. In my opinion, having one sentence per line doesn’t always help and can sound unnatural. I’ve been using ChatGPT to generate these copies, but it might be a good idea to get some help from @stirby.

> I think it'd also be good to reference the template version.

Absolutely, that would be helpful.

Reply:
@stirby It's okay. I will check how to add lists to the email template. Another thing to note is that you wrote this template with the workspace owner in mind, but this email will be sent to the template owner. The purpose of this notification is to track when a template fails.

We might consider changing the "TemplateWorkspaceAutobuildFailed" template to work for this scenario as well as for automatic build failures. 🤔 They are quite similar, in my opinion, and the copy would only need to change slightly. What do you think, @dannykopping?

Reply:
>  wrote this template with the workspace owner in mind

My mistake, fixed below. I would bias towards using one notification template for both cases. 

The following copy is better suited for template administrators.

```
"Build Failed: Workspace '{workspace-name}' using template '{template-name}'"

Hi {UserName}

A workspace using the template {template-name} failed to build.
- Template version: {template-version} 
- Workspace: {workspace-name}
- Build type: {build-type}
- Initiated by: {initiator}

You can debug this workspace using the build logs below or contact the deployment administrator. 
```

Reply:
We can use the same template for both manual or automatic build failures, and just pass in a label to specify which or conditionalize the template.

I'd prefer a table over an unordered list as presented above, but I don't feel strongly about it.

> I will check how to add lists to the email template

It's just markdown, so the way @stirby wrote it above will work.

Reply:
> I'd prefer a table over an unordered list as presented above, but I don't feel strongly about it.

I rarely see tables being used in emails to display information.

What do you think about the following alternative?

```
Hi {UserName},

The workspace {workspace-name} using the template {template-name} (version {template-version}) failed to build.

This issue occurred during the {build-type} phase and was initiated by {initiator}.

You can debug this workspace using the build logs below or contact the deployment administrator. 
```

Reply:
What would `{build-type}` be here? `manual` vs `automatic`?

Reply:
It would be the transition. Bad name, sorry!

Reply:
I used bullets instead of natural text because I imagine people switching between tabs and glancing back at the content. Finding it in a sentence is just more effort. A table would be fine too.

It's not important enough to continue refactoring but I don't see how the latest iteration is an improvement.

![Screenshot 2024-08-15 at 11 52 43 AM](https://github.com/user-attachments/assets/9d4ed583-132a-4346-acde-2f20d9aec5e1)

![Screenshot 2024-08-15 at 11 52 17 AM](https://github.com/user-attachments/assets/97609a15-7fd0-481a-bc7d-4e6d853402aa)

TLDR, I prefer lists/data points over long sentences, but I'll leave it up to you.

Reply:
I personally like how the list looks like 👍 

<img width="1234" alt="Screenshot 2024-08-15 at 15 43 17" src="https://github.com/user-attachments/assets/3d3022ef-7432-4133-a904-263896db4548">